### PR TITLE
Fetch repo from custom go import path

### PR DESCRIPTION
**Pull request by stefanbuck**
The key to get custom import working is in the <meta> tag named `go-import`. When requesting https://k8s.io/kubernetes/pkg/api we know that `k8s.io/kubernetes/pkg/api` points to https://github.com/kubernetes/kubernetes/tree/master/pkg/api.

[http://localhost:3000/q/go/k8s.io/kubernetes/pkg/api](http://localhost:3000/q/go/k8s.io/kubernetes/pkg/api)
[http://localhost:3000/q/go/code.gitea.io/gitea/modules/setting](http://localhost:3000/q/go/code.gitea.io/gitea/modules/setting)

https://github.com/OctoLinker/browser-extension/issues/264

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,11 +4,11 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "start": "node server.js",
+    "start": "node  --harmony server.js",
```
Comment (stefanbuck) [inline package.json]: the harmony flag is need for `async/await`

File: package.json [hunk 2]
```diff
@@ -17,13 +17,15 @@
   },
   "dependencies": {
     "boom": "4.2.0",
+    "find-reachable-urls": "1.0.0",
```
Comment (stefanbuck) [inline package.json]: https://github.com/stefanbuck/find-reachable-urls

File: package.json [hunk 3]
```diff
@@ -17,13 +17,15 @@
...
     "github-url-to-object": "3.0.0",
     "got": "6.7.1",
     "hapi": "16.1.0",
     "is-url": "1.2.2",
     "joi": "10.2.1",
     "json-path": "0.1.3",
     "keen.io": "0.1.3",
+    "lets-get-meta": "2.0.1",
```
Comment (stefanbuck) [inline package.json]: https://github.com/zeke/lets-get-meta/

File: package.json (hunk 1 above)
Comment (josephfrazier) [inline package.json]: Node v7.6.0 was released today with `async/await` support by default. If we upgrade to that, we should be able to remove the `--harmony` flags here and in the test commands below.

**Pull request merged**